tests/unit/test_validation_builder.py

```diff
@@ -242,15 +242,3 @@ def test_custom_query_validation(module_under_test):
     assert (
```

Comment:
Can we delete this test too on line 232? This shouldn't be in the test_validation_builder() and it's not asserting anything useful.